cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -214,6 +220,8 @@ download_binary_and_run_installer() {
       exit 1
     fi
 
+    verify_checksum "$_file" "$_checksum_style" "$_checksum_value"
```

Comment:
Alternately, you can provide a default here, eg `"${_checksum_style:-}"`, so referencing it when undefined isn't fatal.